public/app/features/admin/LdapUserPage.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { PureComponent } from 'react';
+import { hot } from 'react-hot-loader';
+import { connect } from 'react-redux';
+import { css } from 'emotion';
+import { NavModel } from '@grafana/data';
+import Page from '../../core/components/Page/Page';
+import { AlertBox } from '../../core/components/AlertBox/AlertBox';
+import { getNavModel } from '../../core/selectors/navModel';
+import {
+  AppNotificationSeverity,
+  LdapError,
+  LdapUser,
+  StoreState,
+  User,
+  UserSession,
+  SyncInfo,
+  LdapUserSyncInfo,
+} from 'app/types';
+import {
+  clearUserError,
+  loadLdapUserInfo,
+  revokeSession,
+  revokeAllSessions,
+  loadLdapSyncStatus,
+  syncUser,
+} from './state/actions';
+import { LdapUserInfo } from './LdapUserInfo';
+import { getRouteParamsId } from 'app/core/selectors/location';
+import { UserSessions } from './UserSessions';
+import { UserInfo } from './UserInfo';
+import { UserSyncInfo } from './UserSyncInfo';
+
+interface Props {
+  navModel: NavModel;
+  userId: number;
+  user: User;
+  sessions: UserSession[];
+  ldapUser: LdapUser;
+  userError?: LdapError;
+  ldapSyncInfo?: SyncInfo;
+
+  loadLdapUserInfo: typeof loadLdapUserInfo;
+  clearUserError: typeof clearUserError;
+  loadLdapSyncStatus: typeof loadLdapSyncStatus;
+  syncUser: typeof syncUser;
+  revokeSession: typeof revokeSession;
+  revokeAllSessions: typeof revokeAllSessions;
+}
+
+interface State {
+  isLoading: boolean;
+}
+
+export class LdapUserPage extends PureComponent<Props, State> {
+  state = {
+    isLoading: true,
+  };
+
+  async componentDidMount() {
+    const { userId, loadLdapUserInfo, loadLdapSyncStatus } = this.props;
+    try {
+      await loadLdapUserInfo(userId);
+      await loadLdapSyncStatus();
+    } finally {
+      this.setState({ isLoading: false });
+    }
+  }
+
+  onClearUserError = () => {
+    this.props.clearUserError();
+  };
+
+  onSyncUser = () => {
+    const { syncUser, user } = this.props;
+    if (syncUser && user) {
+      syncUser(user.id);
+    }
+  };
+
+  onSessionRevoke = (tokenId: number) => {
+    const { userId, revokeSession } = this.props;
+    revokeSession(tokenId, userId);
+  };
+
+  onAllSessionsRevoke = () => {
+    const { userId, revokeAllSessions } = this.props;
+    revokeAllSessions(userId);
+  };
+
+  render() {
+    const { user, ldapUser, userError, navModel, sessions, ldapSyncInfo } = this.props;
+    const { isLoading } = this.state;
+
+    const userSyncInfo: LdapUserSyncInfo = {};
+    if (ldapSyncInfo) {
+      userSyncInfo.nextSync = ldapSyncInfo.nextSync;
+    }
+    if (user) {
+      userSyncInfo.prevSync = (user as any).updatedAt;
+    }
+
+    const tableStyle = css`
+      margin-bottom: 48px;
```

Comment:
Instead of using fixed values you could use values from `theme.spacing` namespace. See https://github.com/grafana/grafana/blob/master/style_guides/themes.md#using-themes-in-grafanas-react-components on how to use themes.